system/security/shared/SecureUser.hpp

```diff
@@ -200,7 +200,7 @@ class CSecureUser : public CInterface,
 
     virtual CDateTime & getPasswordExpiration(CDateTime& expirationDate){ return expirationDate; }
     virtual bool setPasswordExpiration(CDateTime& expirationDate) { return true; }
-    virtual int getPasswordDaysRemaining() {return -1;}
+    virtual int getPasswordDaysRemaining() {return -2;}//never expires
```

Comment:
There should be some constants in seclib.hpp to make these magic values clearer

e.g.,

const static int PasswordExpired = -1;
const static int PasswordNeverExpires = -2;

Could possibly do as a separate commit though.  If so please open a jira and I'll merge this.